src/modules/java.rs

```diff
@@ -0,0 +1,105 @@
+use std::process::Command;
+
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Creates a module with the current Java version
+///
+/// Will display the Java version if any of the following criteria are met:
+///     - Current directory contains a file with a `.java` extension
```

Comment:
Nitpick: aren't these comments inaccurate now, since we're also scanning for jar/class files and `build.gradle`?

Reply:
Fixed